js/utils.js

```diff
@@ -17,6 +17,55 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
   };
   var computedFieldClashes = [];
 
+  function setLineClamps () {
+    setClampHeight()
+      .then(function (elements) {
+        Superclamp.register(elements);
+        return Promise.resolve(elements);
+      })
+      .then(function (elements) {
+        removeClampHeight(elements);
+      })
+      .catch(function (selector) {
+        console.error('Fail to clamp line in the selector ' + selector);
+      });
+  }
+
+  function setClampHeight (selector) {
```

Comment:
Can you explain again (after so many rounds of changes), why we need to do this, now that we're using Superclamp?

Reply:
@tonytlwu 
As SuperClamp says in the [readme file](https://github.com/makandra/superclamp#usage)

> You want to truncate text to fit inside an element, so define your width/height restrictions on the element, e.g. via CSS. Superclamp will respect an element's padding.

That's why we need to set a fixed height value to the element.